recipes/gflags/bld.bat

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX -DINSTALL_HEADERS=on -DBUILD_SHARED_LIBS=on -DBUILD_TESTING=on
```

Comment:
Not sure if this will do what we want on Windows. Might need to move some files around. Also, we are not specifying the compiler here and maybe we should be doing that. Would be good to have feedback from someone that uses Windows more.